runtime/cfdumper/main.c

```diff
@@ -917,6 +917,17 @@ static void dumpAttribute(J9CfrClassFile* classfile, J9CfrAttribute* attrib, U_3
 			}
 			break;
 
+#if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)
+		case CFR_ATTRIBUTE_Preload:
+			for(i = 0; i < ((J9CfrAttributePreload*)attrib)->numberOfClasses; i++) {
+				U_16 classIndex = ((J9CfrAttributePreload*)attrib)->classes[i];
+				U_16 nameIndex = classfile->constantPool[classIndex].slot1;
+
+				for(j = 0; j < tabLevel + 1; j++) j9tty_printf( PORTLIB, "  ");
```

Comment:
Please put `j9tty_printf()` in the new line inside `{}`.

Reply:
This follows the convention in the file but I agree its not very readable.